scripts/convert_ipynb_to_py.sh

```diff
@@ -1,21 +1,34 @@
 #!/bin/bash
 
+# show commands being executed (for debugging purposes)
+#set -x
```

Comment:
You can remove this line if you aren't keeping it for something
```suggestion
```

Reply:
I'm keeping it for debugging purposes, sometimes I need to un uncomment it to show all the commands being executed.